### ISSUE TITLE
[[ in output when Markdown contains task list items

### ISSUE THREAD

**Issue by yoursunny** [earldouglas/codedown#8]
**Task list items** is an extension of [GitHub Flavored Markdown](https://github.github.com/gfm/#task-list-items-extension-). It looks like this:

    * [X] checked
    * [ ] unchecked
    
    ```lang
    code
    ```

If a Markdown document containing a task list is given to `codedown`, the output incorrectly contains `[` characters.

```js
const codedown = require("codedown");
const lines = [
    "* [X] checked",
    "* [ ] unchecked",
    "",
    "```lang",
    "code",
    "```",
];
const input = lines.join("\n");
console.log(codedown(input, "lang"));
```

Expected output: `code`
Actual output: `[[code`

**Comment by earldouglas** [comment]
@yoursunny Are you still running into this with the latest version of codedown?

I added a test in #18 that passes, so I think it might have been fixed by some other change.

**Comment by yoursunny** [comment]
I believe this bug has been resolved in codedown 2.1.7.

**Comment by earldouglas** [comment]
Thanks, and apologies for the delayed response to this issue!
